types/material-ui/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for material-ui v0.20.0
+// Type definitions for material-ui v0.20.1
```

Comment:
Looks like our lint rules are disabled for this package. Can you enable the lints in our `tslint.json` and fix up the version string to only be a `MAJOR.MINOR` version (e.g. `4.7`), as opposed to including the patch or containing a leading `v` (e.g. `v1.2.3`).

Reply:
Thank you and I'm sorry for slow response. Added dt-header lint rule back, and fixes a few minor lint errors as well.